packages/flame/pubspec.yaml

```diff
@@ -23,7 +23,7 @@ dev_dependencies:
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.24.0"
+  flutter: ">=2.5.0"
```

Comment:
imho this should also be relfected in the changelog :-)

Reply:
done